tensorflow/core/ops/nn_ops.cc

```diff
@@ -632,7 +632,9 @@ REGISTER_OP("_FusedDepthwiseConv2dNative")
     // Attributes for the LeakyRelu ----------------------------------------- //
     .Attr("leakyrelu_alpha: float = 0.2")
     // ---------------------------------------------------------------------- //
+
     .SetShapeFn(shape_inference::DepthwiseConv2DNativeShape);
+
 // --------------------------------------------------------------------------
 REGISTER_OP("Conv3D")
```

Comment:
Ah, I meant new lines around the commented separator line, like this:
```suggestion
    // Attributes for the LeakyRelu ----------------------------------------- //
    .Attr("leakyrelu_alpha: float = 0.2")
    // ---------------------------------------------------------------------- //
    .SetShapeFn(shape_inference::DepthwiseConv2DNativeShape);

// --------------------------------------------------------------------------

REGISTER_OP("Conv3D")
```

Reply:
I misunderstood your point, will address it soon, thanks!